src/orange/orangeinp/ConvexHullFinder.hh

```diff
@@ -263,7 +263,7 @@ auto ConvexHullFinder<T>::is_clockwise(size_type i_prev,
 template<class T>
 size_type ConvexHullFinder<T>::calc_next(size_type i) const
 {
-    return (i + 1) % points_.size();
+    return (i < points_.size() - 1 ? i + 1 : 0);
```

Comment:
Even though we can guarantee `points_` isn't empty it's a good practice to add one to the LHS rather than risk underflowing an unsigned integer
```suggestion
    return (i + 1 < points_.size() ? i + 1 : 0);
```